packages/@react-spectrum/overlays/src/OpenTransition.tsx

```diff
@@ -34,7 +34,7 @@ const OPEN_STATES = {
 export function OpenTransition(props) {
   return (
     <Transition timeout={{enter: 0, exit: 350}} {...props}>
-      {(state) => React.Children.map(props.children, child => React.cloneElement(child, {isOpen: !!OPEN_STATES[state]}))}
+      {(state) => React.Children.map(props.children, child => child && React.cloneElement(child, {isOpen: !!OPEN_STATES[state]}))}
```

Comment:
What was this for specifically?

Reply:
Ah, if I remember correctly this was for when I had `{type !== 'fullscreenTakeover' && <Underlay />}` being passed to Overlay previously and it was breaking since it didn't have the above conditional. Lemme try taking it out now that I don't remove the Underlay anymore

Reply:
Ya so it works without it now, but IMO I feel like we should still keep the `child &&` guard in place because React.Children.map will return null for a child that is conditionally rendered. What do you think?

Reply:
Alternative might be to use React.Children.toArray + .map instead of React.Children.map here?

Reply:
nah that's ok. I just wanted to make sure we weren't covering up a bug. null is a legit child so this should be fine to stay.